### PR TITLE
Add documentation about availability in Debian to README.dst (Closes:…

### PR DESCRIPTION
**Pull request by sdelafond**
… #1253)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -117,6 +117,16 @@ Using dnf:
 
     $ dnf install py3status
 
+Debian
```
Comment (tobes) [inline README.rst]: Debian/Ubuntu?

**Comment by tobes** [comment]
Thanks for this 👍

**Comment by ultrabug** [comment]
Thanks a lot @sdelafond !

**Comment by ultrabug** [comment]
btw, there's a mention for pip and debian users:

> Note: Debian users should use pypi-install from the python-stdeb package instead of pip.

shall we remove it or at least drop the bold font?

**Pull request merged**